File: app/src/main/java/de/uni_marburg/mathematik/ds/serval/view/fragments/DashboardFragment.java
package de.uni_marburg.mathematik.ds.serval.view.fragments;

import de.uni_marburg.mathematik.ds.serval.R;
import de.uni_marburg.mathematik.ds.serval.model.event.Event;

/**
 * Created by thames1990 on 28.08.17.
 */
public class DashboardFragment<T extends Event> extends EventFragment<T> {
    
    @Override
    protected int getLayout() {
        return R.layout.fragment_dashboard;
    }
}

File: app/src/main/java/de/uni_marburg/mathematik/ds/serval/view/fragments/DashboardFragment.kt
package de.uni_marburg.mathematik.ds.serval.view.fragments

import android.os.Bundle
import de.uni_marburg.mathematik.ds.serval.Aardvark
import de.uni_marburg.mathematik.ds.serval.R

class DashboardFragment : BaseFragment() {

    override val layout: Int
        get() = R.layout.fragment_dashboard

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        Aardvark.firebaseAnalytics.setCurrentScreen(activity, getString(R.string.screen_dashboard), null)
    }
}
